Examples/Soto/Sources/main.swift

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import AWSLambdaEvents
+import AWSLambdaRuntime
+import SotoS3
+
+let client = AWSClient()
```

Comment:
please pass the ELG here.

Reply:
ELG ?

Reply:
Do you have an Example, I can't find a way to access the underlying EventLoop on Soto 7. I think everything is hidden now as implementation details

Reply:
yeah. true. both should use the same thanks to `NIOSingletons.posixEventLoogProup`. Consider this resolved.